src/bots.py

```diff
@@ -1,8 +1,8 @@
 import time
-import random
 import pprint
 
 import numpy as np
+import _thread
```

Comment:
what do we do with `_thread`?

Reply:
I don't know why it is there and I have removed it in my next version

Reply:
👍 good. ben is single threaded so i was  wondering.